samples/README_Windows.md

```diff
@@ -0,0 +1,98 @@
+# Building Open Enclave SDK Samples on Windows
+
+All the samples that come with the Open Enclave SDK installation share similar directory structure and build instructions. The section contains general information on how to setup/build/sign/run all samples. It's important that you read information on this page before jumping into any individual sample.
+
+## Common Sample information
+
+### How Sample source code directories were structured
+
+Open Enclave SDK helps developers build enclave applications. An enclave application is partitioned into an untrusted component (called a host) and a trusted component (called an enclave). An enclave is a secure container whose memory (text and data) is protected from access by outside entities, including the host, privileged users, and even the hardware. All functionality that needs to be run in a Trusted Execution Environment (TEE) should be compiled into the enclave binary. The enclave may run in an untrusted environment with the expectation that secrets will not be compromised. A host is a normal user mode application that loads an enclave into its address space before starting interacting with an enclave. 
+
+![Sample components diagram](sampledirstructure.png)
+
+All the samples that come with the Open Enclave SDK installation are all structured into two subdirectories (one for enclave and one for host) accordingly.
+
+| Files/dir        |  contents                                   |
+|:-----------------|---------------------------------------------|
+| enclave        | Files needed for building the sample enclave|
+| host           | Files needed for building the host          |
+
+### Prepare samples
+
+Building samples involves writing files into the working directory. You can do this in the directory in which you have installed your samples.
+You can also copy the samples from the location they were installed to another directory.
+
+```cmd
+xcopy  C:\openenclave\share\openenclave\samples C:\mysample
+```
+
+### How to build and run samples
+
+1. [x64 Native Tools Command Prompt for VS(2017 or 2019)](
+https://docs.microsoft.com/en-us/dotnet/framework/tools/developer-command-prompt-for-vs)
+Normally this is accessible under the `Visual Studio 2019 or 2017` folder in the Start Menu.
+
+2. Set OpenEnclave_DIR to the cmake directory in the Open Enclave SDK installation.
```

Comment:
Formatting: I would suggest wrapping variable names in `` so that they are highlighted like code: `OpenEnclave_DIR`